packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -464,7 +480,10 @@ export async function testProxyConnection(
     return {
       status: statusCode || 0,
       body: body || "",
-      error: e.message || "Failed to connect to Proxy server",
+      error:
+        e?.code === "ECONNREFUSED"
+          ? "Failed to connect to proxy server (ECONNREFUSED)"
+          : e.message,
```

Comment:
Do we want a fallback here in case `e.message` is empty?  Not sure this would ever happen in practice, so maybe not worth worrying about.

Reply:
added one just in case